hikvision/camera_1:
  ros__parameters:
    sensor_hostname: ""
    time_interval: 100

hikvision/camera_2:
  ros__parameters:
    sensor_hostname: ""
    time_interval: 100
    